### PR TITLE
Created WebApi Structure

### PR DESCRIPTION
**Pull request by anudeepannin**

### CODE REVIEW

File: postapi/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
```
Comment (lsingh01) [inline postapi/Controllers/WeatherForecastController.cs]: /WeatherForecastController.cs
Can you remove this file? No need for this controller file, as we need to create empty structure with PostController and CommentController having no actions methods on those.
same comment as @akshayverma91 commented on Praveen's PR.

File: postapi/WeatherForecast.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using System;
```
Comment (lsingh01) [inline postapi/WeatherForecast.cs]: This file also not needed, please remove.

File: .ignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+postapi/bin
+postapi/obj
```
Comment (lsingh01) [inline .ignore]: The filename should be .gitignore and put those bin and obj folder on this file, so that it should not be committed.

**Comment by akshayverma91** [comment]
@anudeepannin Also I don't even find empty structure for story 1

File: postapi/Controllers/WeatherForecastController.cs (hunk 1 above)
Comment (anudeepannin) [reply postapi/Controllers/WeatherForecastController.cs]: Updated the Changes

File: postapi/WeatherForecast.cs (hunk 2 above)
Comment (anudeepannin) [reply postapi/WeatherForecast.cs]: Updated the Changes

File: .ignore (hunk 3 above)
Comment (anudeepannin) [reply .ignore]: Updated the Changes

**Review by 00kailash — APPROVED** [review]

**Pull request merged**